src/com/javabasic/lesson7/GuessWhatNumber.java

```diff
@@ -0,0 +1,61 @@
+package com.javabasic.lesson7;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class GuessWhatNumber {
+
+    static int minRangeValue = 0;
+    static int maxRangeValue = 10;
+
+    final static BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String[] args) throws IOException {
+
+        int computerNumber = getRandomInt();
+        System.out.print("What number did the computer choose? ");
+        int intFromUser = isInt();
+        System.out.println("computerNumber: " + computerNumber);
+
+        if (intFromUser != computerNumber) {
```

Comment:
we can optimize here a little bit.
use one variable for user input instead of two `intFromUser ` and `temp `.
Also, you can use `while` instead of `do while`, in this case, you don't need an additional if statement before the loop.